core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -665,19 +665,6 @@ default AlluxioConfiguration getConfiguration() throws IOException {
    */
   boolean renameRenamableFile(String src, String dst) throws IOException;
 
-  /**
-   * Returns an {@link AlluxioURI} representation for the {@link UnderFileSystem} given a base
-   * UFS URI, and the Alluxio path from the base.
-   *
-   * The default implementation simply concatenates the path to the base URI. This should be
-   * overridden if a subclass needs alternate functionality.
-   *
-   * @param ufsBaseUri the base {@link AlluxioURI} in the ufs
-   * @param alluxioPath the path in Alluxio from the given base
-   * @return the UFS {@link AlluxioURI} representing the Alluxio path
-   */
-  AlluxioURI resolveUri(AlluxioURI ufsBaseUri, String alluxioPath);
```

Comment:
From the comment I can see the original intention of the author. This method could be overriden in extensions. Even not for the overriders, there can be backward compatibility issues if this interface is changed. So maybe the best I can do is to leave it there and mark deprecated.